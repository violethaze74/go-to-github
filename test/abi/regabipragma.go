// runindir -gcflags=-c=1
// +build !windows

// Copyright 2021 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// TODO May delete or adapt this test once regabi is the default

package ignore
