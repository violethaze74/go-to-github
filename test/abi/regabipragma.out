# regabipragma.dir/tmp
tmp/foo.go:17:6: declared function F has register params
# regabipragma.dir
./main.go:21:6: declared function f has register params
./main.go:32:9: called function f has register params
./main.go:33:13: called function tmp.F has register params
